pkgs/by-name/ir/ironbar/package.nix

```diff
@@ -56,16 +64,31 @@ rustPlatform.buildRustPackage rec {
     gtk3
   ];
 
-  preFixup = ''
-    gappsWrapperArgs+=(
+  runtimeDeps = [
+    luajitPackages.lgi
+  ];
+
+  buildNoDefaultFeatures = features != [];
+  buildFeatures = features;
+
+  gappsWrapperArgs = ''
       # Thumbnailers
-      --prefix XDG_DATA_DIRS : "${gdk-pixbuf}/share"
-      --prefix XDG_DATA_DIRS : "${librsvg}/share"
-      --prefix XDG_DATA_DIRS : "${webp-pixbuf-loader}/share"
-      --prefix XDG_DATA_DIRS : "${shared-mime-info}/share"
+          --prefix XDG_DATA_DIRS : "${gdk-pixbuf}/share"
+          --prefix XDG_DATA_DIRS : "${librsvg}/share"
+          --prefix XDG_DATA_DIRS : "${webp-pixbuf-loader}/share"
+          --prefix XDG_DATA_DIRS : "${shared-mime-info}/share"
+
+          # gtk-launch
+          --suffix PATH : "${lib.makeBinPath [ gtk3 ]}"
```

Comment:
Indentation is broken here. Should have 4 leading spaces on each line.

Reply:
I've run nixfmt over the file and it's corrected this, thanks for your suggestion.